src/components/Discover/TrendingTokens.tsx

```diff
@@ -196,56 +203,53 @@ function CategoryFilterButton({
   );
 }
 
-function FriendHolders() {
+function FriendPfp({ pfp_url }: { pfp_url: string }) {
   const backgroundColor = useBackgroundColor('surfacePrimary');
+  return (
+    <FastImage
+      source={{ uri: pfp_url }}
+      style={{
+        height: 12 + 2,
+        width: 12 + 2,
```

Comment:
```suggestion
        height: 14,
        width: 14,
```

Reply:
also can be cleanup after, but a lot of these static styles could be moved to StyleSheets